### PR TITLE
Bring CW721 pointer contract up to spec

### PR DESCRIPTION
**Pull request by dvli2007**
## Describe your changes and provide context

- Switched CwERC721 contract to have all queries that exist on [CW721 Base](https://github.com/CosmWasm/cw-nfts/tree/main/contracts/cw721-base) contract
- Add support for royalties standard (ERC2981 / CW2981)

## Testing performed to validate your change

- Added unit tests to `x/evm/client/wasm/query_test.go` to test out two new EVM queries ("TotalSupply" and "RoyaltyInfo")
- Added tests to `contracts/test/CW721toERC721PointerTest.js` that tests out new query responses

### CODE REVIEW

File: contracts/test/lib.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const ABI = {
         "function symbol() view returns (string)",
         "function totalSupply() view returns (uint256)",
         "function tokenURI(uint256 tokenId) view returns (string)",
+        "function royaltyInfo(uint256 tokenId, utin256 salePrice) view returns (address, uint256)",
```
Comment (stevenlanders) [inline contracts/test/lib.js]:
```suggestion
        "function royaltyInfo(uint256 tokenId, uint256 salePrice) view returns (address, uint256)",
```

**Review by stevenlanders — APPROVED** [review]
one tweak for a lib.js, lgtm

**Review by besated — APPROVED** [review]

File: contracts/test/lib.js (hunk 1 above)
Comment (dvli2007) [reply contracts/test/lib.js]: Good catch. Thanks steven, fixed.

**Comment by dvli2007** [comment]
Closing. https://github.com/sei-protocol/sei-chain/pull/1673 is a copy and was merged in.

**Pull request closed without merging**